src/Compilers/CSharp/Test/Emit/BreakingChanges.cs

```diff
@@ -35,7 +35,7 @@ public void Do()
     }
 }
 ";
-            var comp = CreateCompilationWithMscorlib(text);
+            var comp = CreateStandardCompilation(text);
```

Comment:
Definitely going to have to send an email out about this.  Makes complete sense but need to make sure that going forward we get people to begin to use this is the deafult.